FirebaseInAppMessaging/Swift/Tests/Unit/FirebaseInAppMessaging_APIBuildTests.swift

```diff
@@ -0,0 +1,309 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+
+import FirebaseInAppMessaging
+import FirebaseInAppMessagingSwift
+import SwiftUI
+
+final class FirebaseInAppMessagingSwift_APIBuildTests: XCTestCase {
+  func usage() throws {
+    // MARK: - FirebaseInAppMessaging
+
+    let inAppMessaging = FirebaseInAppMessaging.InAppMessaging.inAppMessaging()
+
+    let _: Bool = inAppMessaging.messageDisplaySuppressed
+    inAppMessaging.messageDisplaySuppressed = true
+
+    // TODO(ncooke3): This should probably be removed in favor of the one in
```

Comment:
IIRC, we support dataCollection is supposed to be controlled both at the Firebase global level and at the individual product. But I only see flags in Performance and Crashlytics? @ryanwilson

Reply:
I made a note to sync with Ryan about this.